components/content_settings/core/common/content_settings_util.cc

```diff
@@ -0,0 +1,91 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/content_settings/core/common/content_settings_util.h"
+
+#include "brave/components/brave_shields/common/brave_shield_constants.h"
+#include "components/content_settings/core/common/content_settings_types.h"
+#include "extensions/common/url_pattern.h"
+
+namespace content_settings {
+
+ContentSetting GetDefaultFromResourceIdentifier(
+    const std::string& resource_identifier,
+    const GURL& primary_url,
+    const GURL& secondary_url) {
+  if (resource_identifier == brave_shields::kAds) {
+    return CONTENT_SETTING_BLOCK;
+  } else if (resource_identifier == brave_shields::kTrackers) {
+    return CONTENT_SETTING_BLOCK;
+  } else if (resource_identifier == brave_shields::kHTTPUpgradableResources) {
+    return CONTENT_SETTING_BLOCK;
+  } else if (resource_identifier == brave_shields::kBraveShields) {
+    return CONTENT_SETTING_ALLOW;
+  } else if (resource_identifier == brave_shields::kReferrers) {
+    return CONTENT_SETTING_BLOCK;
+  } else if (resource_identifier == brave_shields::kCookies) {
+    return secondary_url == GURL("https://firstParty/")
+        ? CONTENT_SETTING_BLOCK
+        : CONTENT_SETTING_ALLOW;
+  }
+  return CONTENT_SETTING_BLOCK;
+}
+
+bool IsWhitelistedCookieException(const GURL& primary_url,
+                                  const GURL& secondary_url,
+                                  bool allow_google_auth) {
+  // Note that there's already an exception for TLD+1, so don't add those here.
+  // Check with the security team before adding exceptions.
+
+  // 1st-party-INdependent whitelist
+  std::vector<URLPattern> fpi_whitelist_patterns = {};
```

Comment:
in the future we may want to get rid of this in favor of something simpler... Bringing here `extensions` doesn't look good

Reply:
this was just moved from another location https://github.com/brave/brave-core/pull/3130/files/6e0deceddd23685bb249bfe98ec0b18a047b63d1#diff-26a6ea1048877a68443a743095591513

Reply:
and I'm not sure that extensions is actually needed, checking now